satellite/metainfo/config.go

```diff
@@ -18,13 +18,26 @@ const (
 	BoltPointerBucket = "pointers"
 )
 
+// RSConfig is a configuration struct that keeps details about default
```

Comment:
since this is identical to the uplink RSConfig struct, I think we should probably use the same one.

Reply:
ok that makes sense

Reply:
type RSConfig struct {
	SatRSConfig uplink.RSConfig `help:"redundancy scheme configuration"`
	Validate bool `help:"toggle flag if rsconfig is enabled" releaseDefault:"true" devDefault:"false"`
}

will this be a good idea?

Reply:
I think based on what Michal said it's fine to keep it how it is